owners/test/reviewer_selection.test.js

```diff
@@ -0,0 +1,253 @@
+/**
+ * Copyright 2019 The AMP HTML Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const sinon = require('sinon');
+const {ReviewerSelection} = require('../src/reviewer_selection');
+const {OwnersRule, OwnersTree} = require('../src/owners');
+
+describe('reviewer selection', () => {
+  const sandbox = sinon.createSandbox();
+  const ownersTree = new OwnersTree();
+
+  const rootDirRule = new OwnersRule('OWNERS.yaml', ['root']);
```

Comment:
Also, since these are only used to create the tree, you can inline them instead of creating a one-time-use const. This way it's easier for me to scroll up and down when reviewing to look at what I expect the results to be :)

Reply:
Whoops, nice catch. These were used at one point, but once `OwnersTree.addRule` was updated to return the subtree, it was no longer necessary. Removed.